runelite-client/src/main/java/net/runelite/client/plugins/itemprices/ItemPricesConfig.java

```diff
@@ -78,4 +78,15 @@ default boolean hideInventory()
 	{
 		return true;
 	}
+
+	@ConfigItem(
+		keyName = "showOnAlt",
+		name = "Show item prices on ALT",
```

Comment:
Why is alt in caps here?

Reply:
Was mostly cause it's called ALT on the keyboard but I changed it.